src/components/__tests__/FlawHistory.spec.ts

```diff
@@ -47,6 +57,11 @@ describe('flawHistory', () => {
       props: {
         history: flaw.history,
       },
+      global: {
+        stubs: {
+          EditableDate: true,
+        },
+      },
     });
     const historyListItem = subject.find('li div');
     expect(historyListItem?.text()).includes('Update Owner:');
```

Comment:
Adding tests for the date range filtering would be awesome

Reply:
Will do that, and also as discussed adding a test for `EditableDate` making sure that it renders and the property placeholder works as expected.